src/dropdown/dropdown.js

```diff
@@ -112,6 +120,40 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
     return self.$element;
   };
 
+  scope.isKeynavEnabled = function() {
+    return keynavEnabled;
+  };
+
+  scope.focusDropdownEntry = function(keyCode) {
+    var elems = self.dropdownMenu ? //If append to body is used.
+      (angular.element(self.dropdownMenu).find('a')) :
+      (angular.element(self.$element).find('ul').eq(0).find('a'));
+
+    switch (keyCode) {
+      case (40): {
+        if ( !angular.isNumber(self.selectedOption)) {
+          self.selectedOption = 0;
+        } else {
+          self.selectedOption = (self.selectedOption === elems.length -1 ?
+                                 self.selectedOption :
+                                 self.selectedOption+1);
+        }
+      }
+      break;
```

Comment:
Of course. 

The official C99 specification says the following about the break statement:
> A break statement terminates execution of the smallest enclosing switch or iteration statement.

So I would vote to put these breaks inside the curly brackets as well.